Utilities/CreateNewExample.py.in

```diff
@@ -166,12 +166,16 @@ def main():
     itk_examples_binary_dir = os.path.abspath('@ITKExamples_BINARY_DIR@')
     itk_examples_build_dir = os.path.abspath(pjoin(itk_examples_binary_dir, os.pardir))
     itk_superbuild_dir = pjoin(itk_examples_build_dir, 'ITK')
-    if not itk_superbuild_dir:
-        print('Error: ITKExamples must be configured using the Superbuild'
-              'functionality to use this script.')
-        sys.exit()
-
-    itk_src = itk_superbuild_dir
+    if not os.path.isdir(itk_superbuild_dir):
+	    # try to see if the itk directory was setup manually and get itk directory from there:
+        itk_dir = os.path.abspath('@ITK_CMAKE_DIR@')
```

Comment:
Two style comments: aligning the comment line with the appropriate indentation helps readability, and we usually capitalize `ITK`, as done [here](https://github.com/InsightSoftwareConsortium/ITKPythonPackage/pull/132#discussion_r330524310).

Reply:
Should be fixed now!
Thanks!

Reply:
@hina-shah thanks. Should have made them before, and even if comment style is not that well documented in the ITK Software guide, I'd start all comments with capital letters, unless a function or other built-in name is involved, and would remove the colon / change it for a period in the end.